docs/schema-conversion-logic.md

```diff
@@ -0,0 +1,98 @@
+# Schema conversion logic
```

Comment:
Do we need to clarify up front that this isn't referring to cassandra/big table schemas?

Reply:
updated, is that better?

Reply:
yep 👍